boards/arm/sama5/giant-board/src/sam_buttons.c

```diff
@@ -0,0 +1,139 @@
+/****************************************************************************
```

Comment:
I think you need to delete this file.  The giant board has not buttons, does it?

Reply:
@patacongo Ok, will delete. I left it because if you do add buttons yourself, you need some code to handle them; but I get that the user should go look for the code themselves and add it to their own application.